src/main/resources/messages.properties

```diff
@@ -1018,7 +1018,7 @@ ethnicity-selection.option1=Hispanic or Latino
 ethnicity-selection.option2=Not Hispanic or Latino
 
 race-selection.title=Race Selection
-race-selection.header=What races do you ({0}) identify with?
+race-selection.header=What race`s do you ({0}) identify with?
```

Comment:
Changing this back. Not sure what happened here, but it's def. not right.

Reply:
Hmm...if we expect only one to be chosen, it should be "race", else it should be "races". "race's" is wrong.